src/Components/LoginForm.js

```diff
@@ -37,23 +39,30 @@ export const  LoginForm = ({ toggleSignup }) =>
           email_id: email.current.value,
           password: password.current.value
       }
-      console.log(payload);
-      AuthenticationService.SignIn(payload).then((response)=>{
+      console.log("data",payload);
+      AuthenticationService.SignIn(payload).then((response) => {
         const message = response.data.Return;
-        console.log(message);
-        window.location.href = '/allprojects';
+        console.log("message", message);
+        const token = response.data.token;
+        // Store the token in local storage
+        localStorage.setItem('token', token);
+        console.log("token1", token);
+        // localStorage.getItem(token);
+      // Redirect the user to a protected route
+        navigate('/allprojects'); // Navigate to /allprojects
       })
       .catch((error)=>{
-        console.log(error.response.data);
-        console.log(error.response.data.error);
-        if(error.response.data.error === "Email is invalid")
-        {
-          setErrorMessageEmail("Invalid Email");
-        }
-        else if(error.response.data.error === "Password is invalid")
-        {
-          setErrorMessagePassword("Invalid Password");
-        }
+        console.error('Login failed:', error);
+        // console.log(error.response.data);    
+        // console.log(error.response.data.error);
+        // if(error.response.data.error === "Email is invalid")
+        // {
+        //   setErrorMessageEmail("Invalid Email");
```

Comment:
Remove unused Commented parts

Reply:
I have just revisited the file, the commented part is for future use. So I need that part for the authentication purpose.

Reply:
Ok make sure that will of use in future..